### ISSUE TITLE
a8n: Small possible UI improvements

### ISSUE THREAD

**Issue by mrnugget** [sourcegraph/sourcegraph#7329]
### Empty burndown chart
<img width="1228" alt="Screen Shot 2019-12-20 at 10 49 17" src="https://user-images.githubusercontent.com/1185253/71248286-c174ca80-231a-11ea-857f-8f2b416f27cb.png">

### "Creation complete" is sticky

Why does this stick around even if I view a newly created Campaign days later?

<img width="763" alt="Screen Shot 2019-12-20 at 10 52 21" src="https://user-images.githubusercontent.com/1185253/71248287-c174ca80-231a-11ea-80d4-97ec7c50654b.png">

### No changesets

Why display the tabs if there are no changesets? I think it would look cleaner if there were not tabs and instead just a "no changesets" and if it's a manual campaign a button "add changesets"

<img width="1325" alt="Screen Shot 2019-12-20 at 10 52 44" src="https://user-images.githubusercontent.com/1185253/71248288-c174ca80-231a-11ea-9727-08b67a7ddf4a.png">

### Campaign Index view

What I want to see here:

- Type of Campaign
- Number of changesets in it
- CreatedAt
- ...

<img width="1205" alt="Screen Shot 2019-12-20 at 10 53 10" src="https://user-images.githubusercontent.com/1185253/71248289-c20d6100-231a-11ea-924c-622d56fc1d82.png">

### Progress Bar

I think it would be great to have proper progress _bar_, instead of just a counter.

**Comment by mrnugget** [comment]
Just discussed this with @eseliger in the Automation sync. These are just things I ran into repeatedly and that I think would improve UI polish by a lot. If you have better ideas than the solutions I hinted at, feel free to implement those :)

(cc @christinaforney )

**Comment by christinaforney** [comment]
I agree with all of @mrnugget's suggestions. In addition on the index view show open/closed status of each campaign.

**Comment by mrnugget** [comment]
Another thing that just confused me is the green icon here to signify that the campaign is still open:

<img width="337" alt="Screen Shot 2020-01-07 at 11 05 02" src="https://user-images.githubusercontent.com/1185253/71887058-9b825f00-313d-11ea-8c1d-5b6e47e0a3d8.png">

What I immediately thought when I saw the green was "this campaign is fully merged, done" but then I realized that it's green because we use green for "open" changesets:

<img width="690" alt="Screen Shot 2020-01-07 at 11 06 56" src="https://user-images.githubusercontent.com/1185253/71887174-d97f8300-313d-11ea-8731-166556a51d98.png">

But since we also use green for "Creation complete" I don't think that is really consistent.

I think we can do one of these:
1. Not use colors for the Campaign icons, but instead enhance the icon with a subicon (checkmark when it's fully merged, gear when open, etc.)
2. Not use colors, but use badges in the Campaigns list and campaign detail header (like GitHub's "Open"/"Closed" badges for issues)
3. Show the current changeset counts in the Campaigns list ("0 open, 18 merged" etc)
